libc/test/src/stdlib/atexit_test.cpp

```diff
@@ -6,7 +6,6 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "hdr/func/_Exit.h"
```

Comment:
looks like you need to include `_Exit` here explicitly (`#include "src/stdlib/_Exit.h`). Applies to both tests.

Reply:
ok will fix it right now